### DISCUSSION TITLE
Windows installing updates, cannot disable?

### DISCUSSION THREAD

**Discussion by fireant21** [Atlas-OS/Atlas#1294] [Help & Support] [answered]
I formatted/installed Atlas with the new 24h2 update, and weeks later just got a Windows Update notification. I swore I had Windows Updates disabled when I configured Atlas. The entire reason I reinstalled was the fact that Windows had not rolled out Recall/AI Spyware *yet*, and wanted my updates disabled to prevent them from installing said tools.

How do I permanently disable updates? I can only delay them 5 weeks at a time in the Windows Update settings. Do I need to format and reinstall W11/Atlas AGAIN?

Edit 1: I Enabled "Remove access to use all Windows Update features" inside Local Group Policy Editor > Administrative Templates > Windows Components > Windows Update >Manage updates offered from windows Server Update Service.

I can still access Windows Update tool and the website, and updates are still listed as "paused for 5 weeks". I will have to see if this holds past the 5 week barrier, but I still want to know why "Do not update" did not hold in the first place.

**Comment by SpyderDoesStuff** [comment, marked as answer]
From what i know atlas only disables automatic updates. Not updates completely. I have no idea how to disable them completely so i suggest looking online for how to do that

**Reply by fireant21** [reply]
OK, I disabled through both Local Group Policy Editor as well as Registry Editor. I still see the options for Windows Updates, including "On pause for 5 weeks" option, I will have to see what happens then.

I asked because my previous 22h2 install of Atlas DID completely remove Windows Update. As in: The "program" to update was gone, I could not search for it, and it was not present in Control Panel or Windows Options. If this does stop Windows from automatically installing anything, thats good enough for me. Again, will see again if anything tries to creep back in.

Thanks kindly.

**Reply by Ast3risk-ops** [reply]
Atlas disables annoying feature updates. Do not fully disable updates as you won't get important security fixes.

**Reply by SpyderDoesStuff** [reply]
atlas DID used to completely remove updates. but that was changed because it was deemed a security risk. which is also why they added the option to keep window defender as well

**Comment by Gari-P** [comment]
Hey, I migrated this year from Windows 7 to Windows 11 and more recently Atlas.

I have a simple question for those who have a fair knowledge in cybersecurity. Have you ever had real issues for not updating Windows 11 or Atlas? I am asking because in nearly 3 decades I NEVER updated my OS. Ok sometimes, once every 3 or 4 years max, you need a new Service Pack to be able to run some programs. And I NEVER got any issue by doing so, my simple rule was to update only when I had to (same with my phones and tablets). I had a very simple free firewall ZoneAlarm; Windows Firewall and Defender were always disable... And I used my computers in every possible way, no limitation. So why would it be any different with Windows 11?

Please only from your personal experience, not something we all read on the Internet.

**Reply by Ast3risk-ops** [reply]
You having used windows 7 for at least a year says volumes about your knowledge of security.

If your PC runs windows 11 like shit use a Linux distribution.

**Reply by SpyderDoesStuff** [reply]
I second this. The lack of knowledge in security is crazy

**Reply by Ast3risk-ops** [reply]
> You having used windows 7 for at least a year says volumes about your knowledge of security.
> 
> If your PC runs windows 11 like shit use a Linux distribution.

Please actually study the most basic aspects of security before asking this stuff. You need an antivirus and a firewall, and you do need updates (Atlas disables the unimportant annoying updates for you so you only get security or important feature updates :wink:).

**Reply by Gari-P** [reply]
Condescending comments, people not able to read carefully, like if I said I was using windows 7 at the moment, like if I wrote that my computer was running like "shit" to use your own words... or that I didn't use a firewall and yes I did study cyber security in my time FYI... it's not about what you read or study but about what you experiment for real.
So that's the only kind of comments I will get here? Some guys acting like angry kids in their mom's basement? I was told that the forum was "nice and friendly".

Anyone who had an actual issue not being at the latest update or just before a patch was issued and would have fixed that? That was the question.

**Comment by amymor** [comment]
Windows Update is required to download and install on Store/Xbox. But I created a script to disable the update completely, it adds a toggle option in the right-click menu on the desktop, so you can easily enable Windows Update if needed.
https://github.com/amymor/1Click-Update
Tested on Windows 10 22h2
You can test it on Windows 11 as well, don't worry about testing as it can restore everything to defaults.

**Reply by Ast3risk-ops** [reply]
It won't show in the win11 context menu, only the "show more options" menu. And again, you'd have to be a moron to disable *all* updates.

**Reply by WarmXR** [reply]
People think disabling all updates is going to make their system better when in reality it's the opposite.

**Reply by Ast3risk-ops** [reply]
> People think disabling all updates is going to make their system better when in reality it's the opposite.

I mean the feature updates Atlas delays are usually useless (overall). I don't get disabling security updates.

**Reply by amymor** [reply]
> you'd have to be a moron to disable _all_ updates.

It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.

**Reply by Ast3risk-ops** [reply]
> > you'd have to be a moron to disable _all_ updates.
> 
> 
> 
> It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.

Atlas disables automatic updates so you get exactly that without a shell extension.

**Reply by SpyderDoesStuff** [reply]
> > you'd have to be a moron to disable _all_ updates.
> 
> It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.

Checking changelogs and bugs and using common sense with downloaded files literally has nothing to do with disabling all windows updates. Its still moronic

**Reply by amymor** [reply]
> > > you'd have to be a moron to disable _all_ updates.
> > 
> > 
> > It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.
> 
> Atlas disables automatic updates so you get exactly that without a shell extension.

But windows update is still connected to ms servers and may spy (who knows?), so where is that extra security/privacy? Just get a good firewall (like simplewall) and check yourself.
So disabling updates in some situations adds more protection for personal data.

**Reply by SpyderDoesStuff** [reply]
> > > > you'd have to be a moron to disable _all_ updates.
> > > 
> > > 
> > > It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.
> > 
> > Atlas disables automatic updates so you get exactly that without a shell extension.
> 
> But windows update is still connected to ms servers and may spy (who knows?), so where is that extra security/privacy? Just get a good firewall (like simplewall) and check yourself.
> So disabling updates in some situations adds more protection for personal data.

It would still take an EXTREME amount of paranoia to throw security out the window and disable them all together

**Reply by Ast3risk-ops** [reply]
> > > > you'd have to be a moron to disable _all_ updates.
> > > 
> > > 
> > > It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.
> > 
> > 
> > Atlas disables automatic updates so you get exactly that without a shell extension.
> 
> But windows update is still connected to ms servers and may spy (who knows?), so where is that extra security/privacy? Just get a good firewall (like simplewall) and check yourself. So disabling updates in some situations adds more protection for personal data.

It connects to the same servers for other purposes, you've done nothing. https://github.com/crazy-max/windowsspyblocker is a program that will automatically block m$ telemetry servers in the windows defender firewall without disabling updates.

**Reply by amymor** [reply]
> > > > > you'd have to be a moron to disable _all_ updates.
> > > > 
> > > > 
> > > > It depends, if you're a moron opening random unknown websites/files then yes, but if you know what you're doing and check the changelog/bugs then no. It's just a toggle option for when you really want to delay that pesky update, it's up to you what you're doing. Let's just say this gives you the right to choose when you want to update.
> > > 
> > > 
> > > Atlas disables automatic updates so you get exactly that without a shell extension.
> > 
> > 
> > But windows update is still connected to ms servers and may spy (who knows?), so where is that extra security/privacy? Just get a good firewall (like simplewall) and check yourself. So disabling updates in some situations adds more protection for personal data.
> 
> It connects to the same servers for other purposes, you've done nothing. https://github.com/crazy-max/windowsspyblocker is a program that will automatically block m$ telemetry servers in the windows defender firewall without disabling updates.

Thanks, but I already block everything via WFP and host file, so disabling update adds an extra layer for me until I want to update or install some games, also less unnecessary processes/services is always better. (Since the WU service runs automatically even when you disable it, there is a scheduled task that enables WU, which can only be disabled with the TI privilege).
